src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -409,8 +419,11 @@ public FormValidation doValidateS3BucketConfig(
             @QueryParameter String customEndpoint,
             @QueryParameter String customSigningRegion) {
         Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+
+        if (FIPS140.useCompliantAlgorithms() && useHttp) {
+            return FormValidation.warning("Cannot use HTTP in FIPS mode.");
```

Comment:
Suggestion: use validation of type `error`

Reply:
You can also reword the message here .
Something like
``` Validation failed as "use Insecure Http" flag is enabled in FIPS mode```

Reply:
> Suggestion: use validation of type `error`
Fran had suggested to use warning (instead of error).

Reply:
> Validation failed as "use Insecure Http" flag is enabled in FIPS mode



Done